### ISSUE TITLE
Do we need a short description?

### ISSUE THREAD

**Issue by aarongustafson** [w3c/manifest-app-info#4]
This is a common key on app stores. Alternately, should we suggest a "safe" maximum length?

**Comment by marcoscaceres** [comment]
Yeah, safe is probably better... the `description` fields is kinda bad for a long description purpose because it doesn't support markup and paragraphs. I wonder if those are best handled "in store".

**Comment by jgw96** [comment]
Love the idea of `description` possibly supporting markup! I wonder if the "safe" max length here may artificially turn "description" into a short description in a way? I feel there should still be a separate field for a short description as there are use cases for both a specific long and short description.

**Comment by marcoscaceres** [comment]
> Love the idea of description possibly supporting markup! 

Sure risky tho, because then the site needs to sanitize it and it gets complicated very quickly: how much markup is allowed, what tags, markdown? etc. 

This reminds me, we need a security consideration that these unrestrained inputs are wide open for XSS attacks on the store front when a manifest is ingested. Similarly, a manifest can attack the JSON parser.

**Comment by marcoscaceres** [comment]
>  I wonder if the "safe" max length here may artificially turn "description" into a short description in a way?

I think stores are in a better position to hide long descriptions. They can just "..." long ones, and provide users with a way of expanding a description.
